DTCoreText.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name         = 'DTCoreText'
-  spec.version      = '1.7.12'
+  spec.version      = '1.8.0'
```

Comment:
this needs to be a 2.0.0 if you really want to remove the old version of the method in DTAttributedTextContentView.h